src/erlcloud_aws.erl

```diff
@@ -781,7 +781,10 @@ service_config( <<"cur">>, Region, Config ) ->
     Config#aws_config{cur_host = Host};
 service_config( <<"application_autoscaling">>, Region, Config ) ->
     Host = service_host(<<"application_autoscaling">>, Region),
-    Config#aws_config{cur_host = Host}.
+    Config#aws_config{application_autoscaling_host = Host};
```

Comment:
Please note a small fix here.

Reply:
wow. so it even even work?

Reply:
I don't think it ever worked. Probably hardly used by anyone.

Reply:
Yeah, we use `application_autoscaling` in our internal DynamoDB table management client, but we write it ourselves in one master configuration for the client's region (along with DynamoDB and DynamoDB Streams host). Nice catch.